stories/Badge.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { Badge, Icon } from '../src/components';
+import '../src/styles';
+import './stories.scss';
+
+const Template = (args) => {
+  return <Badge {...args} />;
+};
+
+export const Default = Template.bind({});
+
+Default.args = {
+  children: 'Some Text',
+  variant: ''
+};
+
+export const Badges: React.FC = () => (
```

Comment:
What do you think about separating each variant in a different story?

Reply:
As you want I followed the Buttons stories structure

Reply:
Done

Reply:
Yes indeed... the Buttons stories is one of the first stories and may need some refinement 
Thanks for your modifications :)